food-bank-app/src/screens/HomeScreen.tsx

```diff
@@ -1,24 +1,144 @@
-import { useAuthContext } from '../contexts/AuthContext'
-import { styles } from '../styles/styles'
 import {
-  Button,
-  Text,
-  View
+  Image,
+  StyleSheet,
+  View,
+  Text
 } from 'react-native'
-import PriorityListItem from '../components/PriorityListItem/PriorityListItem'
+import EventCarousel from '../components/EventCarousel/EventCarousel'
+import PriorityQueue from '../components/PriorityQueueItem/PriorityQueue'
+import EventCard from '../components/EventCard'
+
+const eventCards = [
+  {
+    address: {
+      street: 'Calle El Chaco',
+      exteriorNumber: '3200',
+      interiorNumber: '',
+      zipCode: '43219',
+      state: 'Jalisco',
+      municipality: 'Guadalajara',
+      neighborhood: 'Providencia'
+    },
+    place: 'Bosque Colomos',
+    startDate: new Date(2022, 12, 12),
+    endDate: new Date(2022, 12, 15),
+    startTime: '10:00AM',
+    endTime: '5:00PM',
+    imageUrl: 'https://dynamic-media-cdn.tripadvisor.com/media/photo-o/0e/2f/1c/7b/jardin-japones.jpg?w=1200&h=-1&s=1'
+  },
+  {
+    address: {
+      street: 'asdasa',
+      exteriorNumber: '12',
+      interiorNumber: '123',
+      zipCode: '43219',
+      state: 'Jalisco',
+      municipality: 'Guadalajara',
+      neighborhood: 'centro'
+    },
+    place: 'Bosque Colomos',
+    startDate: new Date(2022, 12, 12),
+    endDate: new Date(2022, 12, 15),
+    startTime: '10:00AM',
+    endTime: '5:00PM',
+    imageUrl: ''
+  }
+]
+
+const displayWelcoMessage = (): string => {
```

Comment:
Aaahhh pero, buena función jaja
```suggestion
const displayWelcomeMessage = (): string => {
```

Reply:
jajajaja

Reply:
Esa dejala jaja suena bien